### ISSUE TITLE
Kernel recipe not found

### ISSUE THREAD

**Issue by uhofemei** [crosswalk-project/meta-crosswalk-embedded#5]
Using the instructions as is, I get 

bitbake core-image-crosswalk-embedded
Loading cache: 100% |##############################################################################################################################################################################| ETA:  00:00:00
Loaded 1301 entries from dependency cache.
ERROR: No recipes available for:
  /data/yocto/meta-crosswalk-embedded/recipes-kernel/linux/linux-yocto_3.19.bbappend

Summary: There was 1 ERROR message shown, returning a non-zero exit code.

**Comment by uhofemei** [comment]
Figured it out. The recipe for kernel 3.19 doesn't exist in the dizzy branch. It comes from master.

**Comment by tiagovignatti** [comment]
@uhofemei yes, you're right! Sorry. We need to update the howto... doing it now.

**Comment by tiagovignatti** [comment]
@uhofemei, howto now should be fixed. Please try it out!
